src/components/dropdownlist.js

```diff
@@ -0,0 +1,49 @@
+import React, { useState } from 'react';
+import 'bootstrap/js/dist/dropdown';
+
+const LanguageDropdown = ({ title }) => {
+  const [local, setLocal] = useState('en_us');
+
+  const handleSelect = (selectedLocal) => {
+    setLocal(selectedLocal);
+  };
+
+  const currentLang = `this page is in ${local}`;
+
+  return (
+    <div className="dropdown">
+      <div>{currentLang}</div>
+      <button
+        className="btn btn-secondary dropdown-toggle"
+        type="button"
+        data-bs-toggle="dropdown"
+        aria-expanded="false"
+      >
+        {local}
+      </button>
+
+      <ul className="dropdown-menu">
+        <li>
+          <button
+            className={`dropdown-item ${local === 'en_us' ? 'active' : ''}`}
+            type="button"
+            onClick={() => handleSelect('en_us')}
+          >
+            en-us
```

Comment:
This should be ENG, see example here
<img width="1383" alt="Screen Shot 2023-07-29 at 4 48 28 AM" src="https://github.com/WilsonLaw23/react-demo/assets/5618294/02beeb18-67b5-4b66-820d-62ee199b86d5">